### PR TITLE
Fix Chrome/Edge versions for showNotification's options.actions parameter

### PR DESCRIPTION
**Pull request by foolip**
Original source: https://github.com/mdn/browser-compat-data/pull/1255

However, both the orignally suggested 47 and the updated 45 are wrong,
and no evidence is given for either version.

This was enabled together with Notifications.maxActions in 48:
https://storage.googleapis.com/chromium-find-releases-static/2a3.html#2a381fb7a46742b05d199c6614624b1bf1432ebb

We can also confirm 48 is correct with this test:
https://mdn-bcd-collector.appspot.com/tests/api/Notification/maxActions

### CODE REVIEW

File: api/ServiceWorkerRegistration.json [hunk 1]
```diff
@@ -652,13 +652,13 @@
...
-                "version_added": "45"
+                "version_added": "48"
               },
               "chrome_android": {
-                "version_added": "45"
+                "version_added": "48"
               },
               "edge": {
-                "version_added": "≤79"
+                "version_added": "18"
```
Comment (foolip) [inline api/ServiceWorkerRegistration.json:661]: This matches `maxActions` which I also tested in Edge 17/18 using https://mdn-bcd-collector.appspot.com/tests/api/Notification/maxActions to confirm it was added in Edge 18.

Since we don't have the source we can't be certain they were added together, but it's very likely I would say.

**Review by queengooborg — APPROVED** [review]
